deploy/helm/scf/assets/operations/set_opensuse_stemcells.yaml

```diff
@@ -1,17 +1,19 @@
 # This ops file sets the openSUSE stemcells.
 
-- type: replace
-  path: /stemcells/alias=default/os
-  value: opensuse-42.3
+- type: remove
```

Comment:
Instead of removing and then adding again below, you can do a replace like:
```
- type: replace
  path: /stemcells/alias=default
  value:
    alias: default
    os: ((...))
    version: ((...))
```

Reply:
I'm removing just the alias (the old stemcell is still listed).  I could totally change things to remove it too, if you prefer.

Reply:
Addressed in be7b1425d579880cc7140b75d24f04d23e7249a1 since the old stemcell isn't useful anyway